tests/common/fixtures/elasticsearch.py

```diff
@@ -22,7 +22,7 @@ def es_client():
                                 conflicts='proceed')
 
 
-@pytest.fixture
+@pytest.fixture(scope="session", autouse=True)
```

Comment:
I don't think we need the change to `es_connect` here any more do we? If we're decided on sticking with `request.es` then we could simply remove the whole fixture and associated code.